tools/misc/get_crowdhuman_id_hw.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+"""Get image shape on CrowdHuman dataset.
+
+Here is an example to run this script.
+
+Example:
+    python tools/misc/get_crowdhuman_id_hw.py ${CONFIG}
+"""
+import argparse
+import json
+import logging
+import os.path as osp
+from multiprocessing import Pool
+
+import mmcv
+from mmengine.config import Config
+from mmengine.fileio import FileClient, dump
+from mmengine.logging import print_log
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Collect image metas')
+    parser.add_argument('config', help='Config file path')
+    parser.add_argument(
+        '--dataset',
+        default='val',
```

Comment:
Do not add default value because two dataset types are both needed to generate hw files.